First part of the solution

bartKarb opened the pull request:
[DEMO LINK](https://bartKarb.github.io/react_todo-app-loading-todos/)

nataliaklonowska commented on src/App.tsx:
```diff
@@ -14,191 +41,22 @@ export const App: React.FC = () => {
+        <TodoHeader />
+        {todos.length > 0 && (
+          <>
+            <TodoList todos={getFilteredTodos(todos, currentFilter)} />
```
you have getFilteredTodos in components folder which is confusing

nataliaklonowska commented on src/App.tsx:
```diff
@@ -14,191 +41,22 @@ export const App: React.FC = () => {
       </div>
+      <ErrorNotification
+        currentError={currentError}
+        setCurrentError={setCurrentError}
```
don't pass setCurrentError as props instead create function to handle this

nataliaklonowska reviewed (changes requested):
remove button looks weird ;)
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/147381375/38745ea5-f5c5-4bb4-b520-85d62a529b91)
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/147381375/140d6690-843b-4fd5-b635-4f39d9f929f9)

choeqq reviewed (changes requested):
@bartKarb did you adjust your solution to comments above? ⬆️

bartKarb commented:
> @bartKarb did you adjust your solution to comments above? ⬆️

Yes, I deleted the highlighted button, moved FilterTodos.tsx file into src folder from components, and changed setCurrentError into handler

nataliaklonowska reviewed (changes requested):
> remove button looks weird ;) ![image](https://private-user-images.githubusercontent.com/147381375/326840336-38745ea5-f5c5-4bb4-b520-85d62a529b91.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUwMDc3MzQsIm5iZiI6MTcxNTAwNzQzNCwicGF0aCI6Ii8xNDczODEzNzUvMzI2ODQwMzM2LTM4NzQ1ZWE1LWY1YzUtNGJiNC1iNTIwLTg1ZDYyYTUyOWI5MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwNlQxNDU3MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lOTE4YmQyMjg4YzljYTk2MTg5NDcyNWM1Y2JkMTVhYjkwZWY3M2RlYzg2MDUzYThmYjBiZTM2NWFhOGYyODNiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7cGCquWqIY3Z51yv--qE6wPLbTTzWaiPwEIXvFGYXPI) ![image](https://private-user-images.githubusercontent.com/147381375/326840565-140d6690-843b-4fd5-b635-4f39d9f929f9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUwMDc3MzQsIm5iZiI6MTcxNTAwNzQzNCwicGF0aCI6Ii8xNDczODEzNzUvMzI2ODQwNTY1LTE0MGQ2NjkwLTg0M2ItNGZkNS1iNjM1LTRmMzlkOWY5MjlmOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwNlQxNDU3MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ODAxNDExNTA5YTc1OTdhYzFlYTlkMTA0OTFhNTk5MzQxNTU1Njg5OTUxMDEzYmIyZDk0ZjliMmM3YTQwMTI5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.phLF247e96PgVtOJ86rBxUxyY3hsoDATUdLSyTCVXdw)

I meant button X to remove todo. It's still looks the same

bartKarb commented:
> > remove button looks weird ;) ![image](https://private-user-images.githubusercontent.com/147381375/326840336-38745ea5-f5c5-4bb4-b520-85d62a529b91.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUwMDc3MzQsIm5iZiI6MTcxNTAwNzQzNCwicGF0aCI6Ii8xNDczODEzNzUvMzI2ODQwMzM2LTM4NzQ1ZWE1LWY1YzUtNGJiNC1iNTIwLTg1ZDYyYTUyOWI5MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwNlQxNDU3MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lOTE4YmQyMjg4YzljYTk2MTg5NDcyNWM1Y2JkMTVhYjkwZWY3M2RlYzg2MDUzYThmYjBiZTM2NWFhOGYyODNiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7cGCquWqIY3Z51yv--qE6wPLbTTzWaiPwEIXvFGYXPI) ![image](https://private-user-images.githubusercontent.com/147381375/326840565-140d6690-843b-4fd5-b635-4f39d9f929f9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUwMDc3MzQsIm5iZiI6MTcxNTAwNzQzNCwicGF0aCI6Ii8xNDczODEzNzUvMzI2ODQwNTY1LTE0MGQ2NjkwLTg0M2ItNGZkNS1iNjM1LTRmMzlkOWY5MjlmOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwNlQxNDU3MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ODAxNDExNTA5YTc1OTdhYzFlYTlkMTA0OTFhNTk5MzQxNTU1Njg5OTUxMDEzYmIyZDk0ZjliMmM3YTQwMTI5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.phLF247e96PgVtOJ86rBxUxyY3hsoDATUdLSyTCVXdw)
> 
> I meant button X to remove todo. It's still looks the same

When I remove this button, I'm getting an error while tests:

`
         should have delete buttons for every todo:
     AssertionError: Timed out retrying after 4000ms: Expected to 
find element: `[data-cy="TodoDelete"]`, but never found it. Queried from:
`
